### PR TITLE
Exclude brands without active help centers

### PR DESCRIPTION
**Pull request by pmgrove**
:koala:

Currently brands that do not have active help centers are displayed in the brand filter. These should be excluded.

/cc @zendesk/quokka

### References
 - Jira link: https://zendesk.atlassian.net/browse/OFFAPPS-558

### Risks
 - None

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -398,7 +398,7 @@
 
     filterBrands: function(brands){
       return _.filter(brands, function(element){
-        return element.active;
+        return element.active && element.has_help_center && element.help_center_state=="enabled";
```
Comment (danielbreves) [inline app.js]: `element.help_center_state == "enabled"`

**Comment by pmgrove** [comment]
:haircut: @danielbreves

File: app.js [hunk 2]
```diff
@@ -398,7 +398,7 @@
 
     filterBrands: function(brands){
       return _.filter(brands, function(element){
-        return element.active;
+        return element.active && element.has_help_center && element.help_center_state === "enabled";
```
Comment (itirkarp) [inline app.js]: Can we skip the `element.has_help_center` check? What does `element.help_center_state` return if it doesn't have a help center?

Comment (pmgrove) [inline app.js]: returns `"help_center_state": "disabled",`

**Comment by danielbreves** [comment]
:+1:

File: app.js (hunk 2 above)
Comment (danielbreves) [inline app.js]: Agree we should be able to skip `has_help_center `

**Comment by pmgrove** [comment]
@itirkarp :haircut:

**Comment by xuebinglee** [comment]
:+1:

**Pull request merged**